docs/api-reference/3d-tiles/tile-3d-header.md

```diff
@@ -40,50 +42,50 @@ Returns `undefined` if `extras` does not exist.
 
 See [Extras in the 3D Tiles specification](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification#specifying-extensions-and-application-specific-extras)
 
-### contentAvailable : Boolean
 
-Determines if the tile has available content to render. `true` if the tile's
-content is ready or if it has expired content that renders while new content loads; otherwise,
-`false`.
+### transform
 
-### contentReady : Boolean
+The local transform of this tile.
+@type {Matrix4}
 
-Determines if the tile's content is ready. This is automatically `true` for
-tile's with empty content.
+### computedTransform
 
-### contentUnloaded : Boolean
+The final computed transform of this tile.
```

Comment:
what does `final` mean? Do we need some more explanation?

Reply:
> what does final mean? Do we need some more explanation?

I moved tile3d-header to wip. It would need too much work to include and is mainly an internal class.